lib/pages/settings/view/settings_page.dart

```diff
@@ -96,6 +111,36 @@ class SettingsPageState extends State<SettingsPage> {
                                 context: context),
                           ),
                         ),
+                        const Divider(),
+                        FlatButton(
+                          key: const ValueKey('ask_permissions'),
+                          onPressed: () => {
+                            if (!authProvider.isVerifiedFromCache &&
+                                !authProvider.isAnonymous)
+                              {
+                                AppToast.show(S
+                                    .of(context)
+                                    .messageEmailNotVerifiedAskPermissions)
+                              }
+                            else if (!authProvider.isAnonymous)
+                              {
+                                Navigator.of(context).push(
+                                    MaterialPageRoute<AskPermissions>(
+                                        builder: (context) =>
+                                            AskPermissions())),
+                              }
+                          },
+                          child: Text(S.of(context).labelAskPermissions,
+                              textAlign: TextAlign.center,
+                              style: authProvider.isAnonymous
+                                  ? Theme.of(context).textTheme.bodyText1.apply(
+                                        color: Theme.of(context).disabledColor,
+                                      )
+                                  : Theme.of(context)
+                                      .textTheme
+                                      .bodyText1
+                                      .apply()),
```

Comment:
`apply()` doesn't do anything here. We only use it when we want to change things in a specific style.